apps/nextjs-website/src/lib/footer.ts

```diff
@@ -1,28 +1,31 @@
 import { FooterProps as FooterPropsEC } from '@pagopa/pagopa-editorial-components/dist/components/Footer';
-import {
-  FooterLinksType,
-  LinkType,
-} from '@pagopa/pagopa-editorial-components/dist/components/Footer/types';
+import * as icons from '@mui/icons-material';
 import { Footer } from './fetch/footer';
+
+type IconNames = keyof typeof icons;
 export type FooterProps = Omit<FooterPropsEC, 'onLanguageChanged'>;
 
-const generateLinks = (
-  section: Footer['data']['attributes']['links_aboutUs'],
-  isSocialLink = false
-): ReadonlyArray<FooterLinksType> =>
-  section.links
-    .filter((link) =>
-      isSocialLink
-        ? link.linkType === 'social' && link.icon
-        : ['internal', 'external'].includes(link.linkType)
-    )
-    .map((link) => ({
-      label: link.text ?? link.href,
-      href: link.href,
-      ariaLabel: link.ariaLabel ?? link.href,
-      linkType: isSocialLink ? 'external' : (link.linkType as LinkType),
-      ...(isSocialLink && { icon: link.icon }),
-    }));
+type Link = Footer['data']['attributes']['links_aboutUs']['links'][0];
+
+const makeLink = ({ text, linkType, ariaLabel, href }: Link) => {
+  const base = {
+    label: text ?? href,
+    href,
+    ariaLabel: ariaLabel ?? href,
+  };
+  return linkType !== 'social' ? [{ ...base, linkType }] : [];
+};
+
+const makeSocialLink = ({ text, icon, linkType, ariaLabel, href }: Link) => {
+  const base = {
+    label: text ?? href,
```

Comment:
socialLinks seems to not care about `label` at all, we can remove it
```suggestion
```
socialLinks seems to not care about `label` at all (it will work without it and it won't render it if present), we can remove it.